.github/workflows/changelog-existence.yml

```diff
@@ -40,10 +44,14 @@ on:
         type: string
         required: true
 
+permissions:
+  contents: read
+  pull-requests: write
+
 jobs:
   changelog_existence:
     name: Check if Changelog Exists
-    if: "!contains(github.event.pull_request.labels.*.name, inputs.skip_label)"
+    if: ${{ !contains(github.event.pull_request.labels.*.name, inputs.skip_label) }}
```

Comment:
This isn't strictly necessary but was causing the editor to mark it as an error.